R/ninjutsu.R

```diff
@@ -0,0 +1,72 @@
+#' Use a new xaringan theme.
+#'
+#' This function clones an xaringan theme for you to use. Include it in a code chunk in the xaringan slide source.
+#'
+#'
+#' @param theme Either a built-in theme shipped with xaringan, or the repository address of the theme in the format of username/repo
+#' @param ref desired git reference. could be a commit SHA, or a branch name. Defauts to master
+#'
+#'
+#' @return The cloned theme
+#' @export
+#'
+#' @examples
+#' # built-in
+#' shadow_clone("kunoichi")
+#' # remote
+#' shadow_clone("tcgriffith/xaringan_theme_example")
+#'
+shadow_clone = function(theme=NULL, ref="master"){
```

Comment:
While the name sounds fancy, I am worried about if it will be descriptive for users.

Reply:
Buuut, it fits the naruto theme!